tests/integration-tests/tests/common/utils.py

```diff
@@ -14,6 +14,8 @@
 import boto3
 from retrying import retry
 
+from pcluster.utils import get_installed_version
+
```

Comment:
Can you please update the query name filter for `centos7` to be the same to the one used in  packer_centos7.json?

Reply:
Done